android/java/org/chromium/base/BraveFeatureList.java

```diff
@@ -33,6 +33,7 @@ public abstract class BraveFeatureList {
     public static final String BRAVE_SHOW_STRICT_FINGERPRINTING_MODE =
             "BraveShowStrictFingerprintingMode";
     public static final String BRAVE_DAY_ZERO_EXPERIMENT = "BraveDayZeroExperiment";
+    public static final String BRAVE_NEW_ANDROID_ONBOARDING = "NewAndroidOnboarding";
```

Comment:
"New onboarding" is very generic. Wouldn't it be clearer to call the feature "AndroidForceDefaultBrowserPrompt"?

Reply:
after talking to @kjozwiak  and @timchilds, we are keeping the flag as `NewAndroidOnboarding`